webpack/webpack.config.js

```diff
@@ -10,28 +15,27 @@ module.exports = {
           name: 'styles',
           test: /\.css$/,
           chunks: 'all',
-          enforce: true,
-        },
-      },
+          enforce: true
+        }
+      }
     },
-    runtimeChunk: true,
+    runtimeChunk: true
+  },
+  entry: {
+    svgSprite: path.resolve(webpackDir, 'svgSprite.js'),
+    css: path.resolve(webpackDir, 'css.js')
   },
-  entry: ["./app.js"],
   module: {
-    rules: [
-      loaders.JSLoader,
-      loaders.CSSLoader,
-      loaders.SVGSpriteLoader,
-    ]
+    rules: [loaders.CSSLoader, loaders.SVGSpriteLoader]
   },
   plugins: [
     plugins.StyleLintPlugin,
     plugins.MiniCssExtractPlugin,
     plugins.ImageminPlugin,
-    plugins.SpriteLoaderPlugin,
+    plugins.SpriteLoaderPlugin
   ],
   output: {
-    path: path.resolve("./dist"),
-    filename: "js/[name].bundle.js"
-  },
+    path: distDir,
+    filename: 'remove/[name].js'
```

Comment:
Another question: What's the "remove" thing here?

Reply:
Oh, it's a directory I want to remove using webpack. It has to be created, but apparently webpack has a cleanup mechanism. I'll work on that.